apps/ejabberd/src/mod_mongoose_admin.erl

```diff
@@ -0,0 +1,234 @@
+-module(mod_mongoose_admin).
+-author('bartlomiej.gorny@erlang-solutions.com').
+
+-behaviour(gen_mod).
+
+-export([start/0, stop/0,
+         start/2, stop/1,
+         register/3,
+         unregister/2,
+         registered_commands/0,
+         registered_users/1,
+         change_user_password/3,
+         list_sessions/1,
+         kick_session/3,
+         get_recent_messages/3,
+         send_message/3
+        ]).
+
+-include("ejabberd.hrl").
+-include("jlib.hrl").
+
+start() ->
+    mongoose_commands:register(commands()).
```

Comment:
OK thanks, I guess this branch will be called https://github.com/esl/MongooseIM/blob/master/apps/ejabberd/src/mongoose_commands.erl#L530